### PR TITLE
Fixes for the nightly release process

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

Move run resources to a dedicated folder, so they are not applied
by mistake when doing `kubectl apply -f tekton`.
Make the bucket resource name unique so that it does not conflict.
Point the bucket directly into the triggers folder, to avoid
downloading the releases of all tekton projects - making the release
process more separated from other projects release process, so we
don't have an issue if the releases run in parallel.
Remove `triggers` from the path when copying to the bucket.

Add a cloudevent resource to trigger post release services (log copy).
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [n/a] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```
Describe any user facing changes here, or delete this block.

Examples of user facing changes:
- API changes
- Bug fixes
- Any changes in behavior

```

### CODE REVIEW

**Comment by dibyom** [comment]
Looks like some indentation issues in the resources file; otherwise LGTM

File: tekton/resources.yaml [hunk 1]
```diff
@@ -51,5 +51,15 @@ metadata:
 spec:
   type: image
   params:
-   - name: url
-     value: cmd/eventlistenersink # Registry is provided via parameter, this is a hack see #569
\ No newline at end of file
+  - name: url
+    value: cmd/eventlistenersink  # Registry is provided via parameter, this is a hack see #569
+# ---
```
Comment (dibyom) [inline tekton/resources.yaml]: Was this meant to be commented out?

Comment (afrittoli) [reply tekton/resources.yaml]: Yeah, the URL of the sink there is cluster specific so I didn't want to include a resource that would actually be applied with a broken url

**Pull request merged**